### ISSUE TITLE
Private PvP arenas

### ISSUE THREAD

**Issue by oraserat** [Matmorcat/Cobbleforest#31]
Private PvP arena
The purpose of these arenas could be used for improv fighting, not just hand to hand ect Snowball fights
Perfect if you want to host an uninterrupted battle instead of being intruded on in AB or WAR
Should be fully customisable using a survival inventory 
Drop items on death and keep inventory should be an option

Arena (suggested) sizes and prices

25x25 1000p (for those small meet ups) 10 players max
50x50 2000p (for a decent game)  25 players max
75x75 3000p (for a large meet up) unlimited players
100x100 4000p (for a full on large battle) unlimited players

Let me know what you think

**Comment by Matmorcat** [comment]
Possibly. It couldn't be done in Build. It would have to be done in the PvP world.

**Comment by oraserat** [comment]
Of course, and certain rules would have to be put in place about what blocks you can use to customise the arena with so no one can set traps.
